great_expectations/datasource/fluent/batch_request.py

```diff
@@ -121,6 +121,8 @@ def __setattr__(self, key, val):
 
     @pydantic.validator("options", pre=True)
     def _validate_options(cls, options) -> BatchRequestOptions:
+        if options is None:
```

Comment:
this fixes a bug where the pre-validator would raise an error if options was initialized to `None`

Reply:
@donaldheppner I thought that you already fixed this by adding a `pydantic.dataclass` default for the options (with the `dict` type), which we did as part of making `Checkpoint` support FDS.  Thus, is this really needed here?  What errors are we seeing that would indicate that this is a bug?  Thanks.

Reply:
the pre-validator made it impossible to assign None to this field without erroring (which is a common initialization scenario)